swagger/api.spec.yaml

```
@@ -1988,6 +1995,31 @@ components:
             type: array
             items:
               $ref: '#/components/schemas/GroupOutWithHostCount'
+    AccountStalenessIn:
+      title: Account Staleness In
+      description: >-
+        Data of a single account staleness.
+      type: object
+      allOf:
```

Should use `anyOf` if one or more of them is required
```suggestion
      anyOf:
```